src/api.js

```diff
@@ -14,25 +14,34 @@ axios.defaults.withCredentials = true
 // Constants
 const BACKEND_URL = process.env.REACT_APP_BACKEND_URL
 
-const getDirectoryFile = async (siteName, folderName) => {
+const getDirectoryFile = async (siteName, folderName, errorCallback) => {
     try {
         return await axios.get(`${BACKEND_URL}/sites/${siteName}/collections/${folderName}/pages/collection.yml`);
     } catch (err) {
         if (err.response && err.response.status === 404) {
-            throw {
-                status: 404,
-                message: err.response.data.message,
-            }
+            errorCallback()
```

Comment:
As discussed offline, we can consider using React Query's onSuccess, onSettled and onError calls to handle these responses directly in Folders.jsx, since many of these responses seem very specific to reordering.

Reply:
Resolved with [05fc9d4](https://github.com/isomerpages/isomercms-frontend/pull/364/commits/05fc9d4a4836c485a3f9ebb04e1c2aea71cda688)